.github/workflows/neon-preview-branches-for-pull-requests.yml

```diff
@@ -12,15 +12,15 @@ on:
           A parameter to create a new branch. 
```

Comment:
descriptions must be updated and have the info what are the default values

Reply:
Descriptions are taken from the corresponding action parameters description - https://github.com/neondatabase/create-branch-action/blob/main/action.yml#L28

But now I've updated them with other information.